src/stan/lang/generator/generate_real_var_type.hpp

```diff
@@ -21,7 +21,7 @@ namespace stan {
                                 bool has_var,
                                 bool is_var_context,
                                 std::ostream& o) {
-      if (is_fun_origin(vo))
+      if (vo.is_fun_origin())
```

Comment:
I like this much better.  I think we should be moving to more of this OO style where appropriate.